content/source/docs/enterprise/getting-started/policies.html.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: "enterprise2"
+page_title: "Managing Sentinel Policies - Getting Started - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-started-policies"
+---
+
+# Creating and Managing Terraform Policies
+
+**Prerequisites:** Before starting this guide, make sure you've [sucessfully completed a run](./runs.html).
+
+## About Sentinel Policies
+
+_Policies_ in TFE are composed of [Sentinel policies](../sentinel/index.html) with some extra features. [Sentinel](https://www.hashicorp.com/sentinel) is an embedded policy-as-code framework integrated with the HashiCorp Enterprise products. It enables fine-grained, logic-based policy decisions, and can be extended to use information from external sources. Within TFE, you can use Sentinel to apply checks to your runs.
+
+A policy consists of:
+
+- The Sentinel policy code
+- An enforcement mode that changes how a policy affects the run lifecycle
+
+Currently, policies in an organization apply to **all** workspaces. You'll want to consider that when writing your policies so they won't fail unrelated workspace runs or configurations. In the future, TFE will provide a way to manage policies so you can apply them per-workspace.
+
+## Creating a Policy
+
+First, make sure you're viewing the organization settings. If you're still on a run page (or any other page), click the "Settings" button in the top navigation bar.
+
+![navigate](./images/policy-navigate.png)
+
+The policy list shows all of the policies you have access to; if you haven't created any, it's empty.
+
+To create your first policy, click the "Create new policy" button in the upper right.
+
+![new button](./images/policy-new-button.png)
+
+On the "Create a new Policy" page, you need to enter at least two items: a policy name, and the policy code. When you've finished, click the "Create Policy" button.
```

Comment:
> the "Create a new Policy" page

Looks like the page is actually titled "Create new Sentinel Policy", as per the screenshot?